app/policies/application_policy.rb

```diff
@@ -13,10 +13,6 @@ def initialize(user, record)
     @record = record
   end
 
-  def index?
-    user.admin?
-  end
-
```

Comment:
Why remove this?
It seems a bit out of place in this PR

Reply:
I changed this because I added `verify_authorized` and `verify_policy scoped`, following the second example here:
https://github.com/varvet/pundit/tree/d7c2e6e3bc1f5526f220a729090249635a7453d8#ensuring-policies-and-scopes-are-used

As the example skips the index for `verify_authorized`, I got the impression that it was not preferable to both authorize and scope for indexes.

I realize now that not having a policy method for `index` will cause some issues though. It will be difficult to create conditional links in the navigation, and it's also not great UX if you are technically able to see an index page but with nothing in it - even if there's no link for the page.

How would you feel about having something like the following instead?:
```ruby
def index?
  Scope.new(user, record) != record.none
end
```

It would allow us to have the logic in the scope, instead of spread across scope and `index?`.

Reply:
After having given this some more thought I think the solution I proposed might be in the "too clever" territory. For now I think it's best to keep things simple and easy to understand.